### DISCUSSION TITLE
ElasticSearch using ***a lot*** of RAM

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [misskey-dev/misskey#8277] [Q&A]
ElasticSearch is using 17GB of RAM, and it's only for Misskey. Is that normal?

**Comment by ThatOneCalculator** [comment]
haha #5138 go brrrrrrr

**Comment by Johann150** [comment]
ElasticSearch is an optional dependency, so if you are struggling with it you do not have to use it. I think personally that the search works fine without it.

**Reply by ThatOneCalculator** [reply]
I mean yea, but 17GB of ram is kinda insane.
